src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -31,7 +31,7 @@ Do some of your requests communicate with services that are still using standard
 To create a new trace observer if you're using New Relic APM agents or third-party integrations:
 
 1. Go to [one.newrelic.com](https://one.newrelic.com) and click **Apps**, and then under **Your apps**, click **New Relic Edge**.
-2. Select an account in the upper-left dropdown. If you have access to multiple accounts, make sure you're in the account where you want Infinite Tracing enabled. Since Tracing is a cross-account feature, the trace observers are limited to one observer per region per [account family](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure/).
+2. Select an account in the upper-left dropdown. If you have access to multiple accounts, make sure you're in the account where you want Infinite Tracing enabled. If you can't add an observer, it's likely due to there being only [one observer allowed per region per account family](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#infinite-architecture).
```

Comment:
Consider simplifying like this: "...it's probably because only one observer is allowed per region, per account family."